packages/next/client/image.tsx

```diff
@@ -396,6 +448,11 @@ export default function Image({
   // it's too late for preloads
   const shouldPreload = priority && typeof window === 'undefined'
 
+  if (unsized) {
```

Comment:
shouldn't it be `if(layout === 'fill')` here?

Reply:
Nope, this branch is for backwards compatibility with `unsized`.

Reply:
Got it :+1: